res/controllers/Hercules DJ Console 4-Mx.midi.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version='1.0' encoding='utf-8'?>
 <MixxxControllerPreset mixxxVersion="2.0.0+" schemaVersion="1">
```

Comment:
Oh I didn't notice the original had (2.0) in the name. I'd remove it from the name then.

We need to do a better job of surfacing "mixxxVersion" in the UI :) currently it does nothing IIRC.